### PR TITLE
Change relative date text

### LINKED ISSUE #4: Change relative date text
`in 4 days` instead of `4 days from now`

### PR DESCRIPTION
**Pull request by SeanAverS**
Fixes #4 

- Change `x days from now` to `in x days` on line 122

### CODE REVIEW

File: src/pickleball.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ function checkPickleballWeather(latitude, longitude) {
         } else if (dayDifference === 1) {
           relativeDay = "Tomorrow";
         } else {
-          relativeDay = dayDifference + " days from now";
+          relativeDay = "in " + dayDifference + " days";
```
Comment (stanchanpsu) [inline src/pickleball.js]:
```suggestion
          relativeDay = "In " + dayDifference + " days";

```

**Review by stanchanpsu — APPROVED** [review]

**Comment by stanchanpsu** [comment]
Thanks for the PR! Feel free to merge.

Out of curiosity, how did you find my repo/issue?

**Comment by SeanAverS** [comment]
Thanks for approving @stanchanpsu 😄 

I found your issue on https://goodfirstissues.com. Regarding merging I think you have to do it since I don't have write access.

**Comment by stanchanpsu** [comment]
> Thanks for approving @stanchanpsu 😄
> 
> I found your issue on https://goodfirstissues.com. Regarding merging I think you have to do it since I don't have write access.

Well thank you so much. I'll merge and deploy now.

**Pull request merged**